content/ops/deploying-the-admin-ui.md

```diff
@@ -80,4 +64,4 @@ Or...
 
 #### Operation
 
-Access the Admin UI at the uri specified in the deployment manifest. By default this is `https://admin.YOUR-SYSTEM-DOMAIN` as generated by `meta.subdomain "." properties.system_domain` in the `templates/admin-ui-deployment.yml`.
+Access the Admin UI at the uri specified in the deployment manifest. By default this is `https://admin.YOUR-SYSTEM-DOMAIN` as generated by `meta.subdomain "." properties.system_domain` in the `templates/admin-ui-deployment.yml`.  If you did not change this setting, try `https://admin.bosh-lite.com`.
```

Comment:
This could be merged or just depreciated in favor of  https://github.com/cloudfoundry-community/admin-ui-boshrelease/blob/master/README.md. ,My impression is that occasionally we either need an upstream release prior to its official release, or may lag a release if it is newer and untested, which was the rationale for this MD.

We could merge this PR and then raise an issue to review if it is still value add?